File: IDAUtil/Model/Properties/TcodeProperty/VA02Obj/OrderObjects/Switches/SwitchesSapOrderProperty.cs
using System.Collections.Generic;

namespace IDAUtil {
    public class SwitchesSapOrderProperty {
        public string salesOrg { get; set; }
        public int orderNumber { get; set; }
        public int soldTo { get; set; }

        //this is the list of switch objects that belong to the same order 
        public List<SwitchesSapLineProperty> lineDetails { get; set; }

        public SwitchesSapOrderProperty() {
        }
    }

    public class SwitchesSapLineProperty {
        public int oldSku { get; set; }
        public int lineNumber { get; set; }
        public bool isSameBarcode { get; set; }
        public string reason { get; set; }
        public int newSku { get; set; }

        public SwitchesSapLineProperty() {
        }
    }
}

File: IDAUtil/Model/Properties/TcodeProperty/VA02Obj/OrderObjects/Switches/SwitchesSapOrderProperty.vb
Public Class SwitchesSapOrderProperty
    Public Property salesOrg As String
    Public Property order As Long
    Public Property soldTo As Long
    Public Property lineDetails As List(Of SwitchesSapLineProperty)

    Public Sub New()
    End Sub

End Class

Public Class SwitchesSapLineProperty
    Public Property oldSku As Long
    Public Property lineNumber As Long
    Public Property isSameBarcode As Boolean
    Public Property reason As String
    Public Property newSku As Long

    Public Sub New()
    End Sub

End Class